File: Difficulty: Basic/Largest Element in Array/largest-element-in-array.cpp
class Solution {
  public:
    void maximum(int i, vector<int> &arr, int &maxi){
        if(i>=arr.size()) return;
        
        maxi = max(maxi, arr[i]);
        maximum(i+1, arr, maxi);
    }
    int largest(vector<int> &arr) {
        // code here
        int maxi = INT_MIN;
        maximum(0, arr, maxi);
        
        return maxi;
    }
};

File: Difficulty: Basic/Largest Element in Array/largest-element-in-array.js

class Solution {
    /**
    * @param number[] arr

    * @returns number
    */
    largest(arr) {
        // code here
        let maximum = 0;
        for(let i=0; i<arr.length; i++){
            if(arr[i]>maximum) maximum = arr[i];
        }
        
        return maximum;
    }
}
